app/models/user.rb

```diff
@@ -14,4 +14,36 @@ class User < ApplicationRecord
   def email_downcase
     self.email = email.downcase
   end
+
+  attr_accessor :remember_token
+
+  class << self
+    def User.digest string
```

Comment:
```suggestion
    def digest string
```
code ko self check nè